baton-maven-plugin/src/test/resources/specifications/migration-json.feature

```diff
@@ -2,38 +2,41 @@
-      | migrateBar | org.technologybrewery.baton.DoesNotExistMigration |
+    | group     | name       | implementation                                    |
+    | Foo       | migrateFoo | org.technologybrewery.baton.NoOpMigration         |
+    | Bar       | migrateBar | org.technologybrewery.baton.DoesNotExistMigration |
 
   Scenario Outline: load a migrations file with optional description field
-    Given a migration with required fields as well as "<description>"
+    Given a group "<group>"
+    And a migration with required fields as well as "<description>" for group "<group>"
     When the configuration is read
-    Then a valid migration is available with "<description>"
+    Then a valid migration is available with "<description>" for group "<group>"
 
     Examples:
-      | description                   |
-      | This migration helps with ABC |
-      | This migration helps with XYZ |
+     | group | description                   |
+     | Foo   | This migration helps with ABC |
+     | Bar   | This migration helps with XYZ |
 
   Scenario Outline: load a migrations file with a file set
-    Given a valid migration with the file set definition of "<directory>", "<includes>", "<excludes>", "<followSymLinks>"
+    Given a group "<group>"
+    And a valid migration with the file set definition of "<directory>", "<includes>", "<excludes>", "<followSymLinks>" for group "<group>"
     When the configuration is read
-    Then a valid migration is available as "<directory>", "<includes>", "<excludes>", "<followSymLinks>"
+    Then a valid migration is available as "<directory>", "<includes>", "<excludes>", "<followSymLinks>" for group "<group>"
 
     Examples:
-      | directory                | includes     | excludes      | followSymLinks |
-      | ./src/my-random-location |              |               |                |
-      | ./src/main/resources     |              |               |                |
-      |                          | **/*.foo     |               |                |
-      |                          | *.toml, *.py |               |                |
-      |                          |              | **/*          |                |
-      |                          |              | *.xml, *.java |                |
-      |                          |              |               | true           |
-      |                          |              |               | false          |
-      |                          |              |               |                |
\ No newline at end of file
+    | group    | directory                | includes     | excludes      | followSymLinks |
+    | Foo      | ./src/my-random-location |              |               |                |
+    | Bar      | ./src/main/resources     |              |               |                |
+    | Foo_Bar  |                          | **/*.foo     |               |                |
```

Comment:
A: same comment about underscores vs. dashes.  Maven typically prefers dashes - it's really easy to mix them, so staying super consistent will help mitigate that.

Reply:
Fixed.